vehicle/OVMS.V3/components/vehicle_renaultzoe/docs/index.rst

```diff
@@ -37,4 +37,11 @@ Others
 After selecting the Renault Zoe/Kangoo vehicle module, the corresponding settings have to be made in the web interface via the "Renault Zoe/Kangoo" menu under "Features":
 
 .. image:: Features.png
-    :width: 480px
\ No newline at end of file
+    :width: 480px
+
+
+----------
+Using Cabin Pre-heat/cool
+----------
+
+Only 5 Minutes Booster are Implementet white Climatecontrol on or Homelink 1
```

Comment:
This is hardly readable or understandable… please clarify and fix spelling. If I understand you correctly, this also contradicts the much broader claim you added to "changes.txt".